### PR TITLE
Patch python3

### LINKED ISSUE #63: Ditch Python2
Let's ditch python2 and adopt python3. ;)

### PR DESCRIPTION
**Pull request by Vipul-Bajaj**
Migrated to python3. Resolves #63

### CODE REVIEW

**Comment by Vipul-Bajaj** [comment]
Hi @shubhodeep9 ,
Does the code have some bug, as I see it has been pending review  for many days?
Do let me know, if something is wrong.

File: hackr/crypto.py [hunk 1]
```diff
@@ -62,7 +64,7 @@ def bcrypt(self):
             f = open(self.wordlist, "r")
             for i in f:
                 plain_text = i.rstrip('\n')
-                if bcrypt.checkpw(plain_text, self.hash):
+                if bcrypt.checkpw(plain_text.encode('utf-8'), self.hash):
```
Comment (shubhodeep9) [inline hackr/crypto.py:67]: particular you added this?
is it particular to `python3`?

**Review by shubhodeep9 — CHANGES_REQUESTED** [review]
Have you covered all modules?

File: hackr/crypto.py (hunk 1 above)
Comment (Vipul-Bajaj) [reply hackr/crypto.py:67]: Actually, I have made changes in almost all the files.
Yes, the changes are specific to Python3.I have covered all the modules,I ran all the commands you have mentioned in the ReadMe and all worked fine and also tested functions individually.
If something breaks, please let me know.

**Review by Vipul-Bajaj — COMMENTED** [review]
Nothing to change

**Pull request closed without merging**